src/Voron/Impl/LowLevelTransaction.cs

```diff
@@ -232,7 +233,7 @@ public LowLevelTransaction(StorageEnvironment env, long id, TransactionPersisten
             _journal = env.Journal;
             _id = id;
             _freeSpaceHandling = freeSpaceHandling;
-            _allocator = context ?? new ByteStringContext(LowMemoryFlag.None);
+            _allocator = context ?? new ByteStringContext(SharedMultipleUseFlag.AlwaysLow);
             _disposeAllocator = context == null;
```

Comment:
Is it always ```.AlwaysLow```? Does it make sense to have a different construction?

Reply:
We can't guarantee it will be always low; we couldn't either with the previous construction. Do you have any better idea in mind?

Reply:
Not really, I was just wondering if it would make sense to call that differently. @ayende what do you think?

Reply:
Yes, `None` or something that say `WillNeverBeSet` will be a better option here.
`AlwaysLow` made me think it was related to the low memory handling.

Reply:
Renamed it to None and added a comment.